sections/appendix/applications.tex

```diff
@@ -0,0 +1,22 @@
+\subsection{Applications of our framework}\label{sec:appendix_applications}
+There are a variety of different applications, which topic modeling can be used for. 
+\citet{Probabilistic_Topic_Models} describes many different purposes for topic modeling, like exploring the history of news articles over time.
+The \gls{lda} model, created by \citet{blei2003latent}, has seen many extension over the years to try an improve the generality of the model.
+Normally \gls{lda} works by inferring hidden topic structure, which is based on two topic distribution, namely document-topic and topic-word distributions. 
+An extension to the \gls{lda} is the Author-Topic model, created by \citet{author_topic_2012}, which creates a relationship between the author meta-information and the corpus.
+This idea of taking meta-information into account seems to be overlooked, even though a few papers have touched upon this area, the application possibilities for these kind of meta-information integration are endless.

```

Comment:
We begin by saying there are many different application, but then we don't talk about the applications, we talk about how lda works and what extensions have been made, etc.
Line 3-7 seems kinda out of place / not sure what they are trying to do.
We do mention application of exploring something over time, but we don't do that, so I'm not sure why we are pointing it out.